scripts/common.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const np = require('path');
+const child_process = require('child_process');
+
+/**
+ * Returns an object where
+ *  key = path to project root
+ *  value = parsed package.json
+ */
+exports.findProjects = function findProjects(rootDir, files = {}) {
+  for (let file of fs.readdirSync(rootDir)) {
+    let projectPath = np.join(rootDir, file);
+    const stats = fs.statSync(projectPath);
+    if (stats && stats.isDirectory()) {
+      let packagePath = np.join(projectPath, 'package.json');
+      if (fs.existsSync(packagePath)) {
+        let package = JSON.parse(fs.readFileSync(packagePath).toString());
```

Comment:
I don't know why package is a keyword but eslint complains: `Parsing error: Unexpected reserved word 'package'
` (also for the occurrences in the other scripts)

Reply:
Fixed. Renamed them all to pack.

Reply:
@mischnic Is this good now?